amoni/api.py

```diff
@@ -22,11 +23,35 @@
 
 __version__ = "0.0.13"
 
-COOKIECUTTER_URL = "https://github.com/anvilistas/amoni-cookiecutter.git"
+COOKIECUTTER_URL = os.environ.get(
+    "AMONI_COOKIECUTTER_PATH", "https://github.com/anvilistas/amoni-cookiecutter.git"
+)
 ANVIL_CONFIG_FILE = Path("app", "config.yaml")
 TABLE_STUB_FILE = Path("anvil-stubs", "tables", "app_tables.pyi")
 
 
+def get_ports() -> Tuple[str, str, str]:
+    """Get the configured ports and origin URL for app and database servers
+
+    Returns
+    -------
+    Tuple[str, str, str]
+        A tuple containing (app_port, db_port, origin_url)
+    """
+    # Look for .env in current directory
+    if not load_dotenv(dotenv_path=".env"):
+        print(
```

Comment:
Rather than using a raw `print`, this should be using the echo module - and probably needs a new `echo.warn` function to be added.